src/main/webapp/pages/accessionvalidationrange/accessionValidationRange.jsp

```diff
@@ -104,6 +104,13 @@ function /*void*/ enableDisableCheckboxes( matchedElement, groupingNumber ){
 	jQuery("selectAllAccept").checked = false;
 }
 
+function criticalResultCheck( index ){
+	var result = jQuery("#resultId_" + index).val();
+	if((result >= 0) && (result <= 10) || (result >= 95)){
```

Comment:
these numbers should not be hardcoded. They need to be pulled from the critical range.